cmd/main.go

```diff
@@ -2,43 +2,52 @@ package main
 
 import (
 	"flag"
+	"net/http"
+	"os"
+	"regexp"
+
 	"github.com/ViaQ/logerr/log"
+	"github.com/fsnotify/fsnotify"
 	"github.com/log-file-metric-exporter/pkg/symnotify"
 	"github.com/prometheus/client_golang/prometheus"
 	"github.com/prometheus/client_golang/prometheus/promhttp"
-	"net/http"
-	"os"
-	"regexp"
 )
 
 var (
-	verbosity int = 0
-	//Reference regexp https://github.com/fabric8io/fluent-plugin-kubernetes_metadata_filter/blob/master/lib/fluent/plugin/filter_kubernetes_metadata.rb#L56, https://github.com/kubernetes/kubernetes/blob/release-1.6/pkg/kubelet/dockertools/docker.go
-	//compile k8 logfilepathname pattern
-	kubernetesregexpCompiled = regexp.MustCompile(`.var.log.containers.([a-z0-9][-a-z0-9]*[a-z0-9])_([^_]+)_(.+)-([a-z0-9]{64})\.log$`)
+	verbosity int = 1
+	logDir        = "/var/log/pods"
+	logFile       = regexp.MustCompile(`/([a-z0-9-]+)_([a-z0-9-]+)_([a-f0-9-]+)/([a-z0-9-]+)/.*\.log`)
```

Comment:
We allow you to pass the logdir but not the logfile.  I realize unlikely to change soon but what happens if they vary? The pattern for containerDir is different from podDir

Reply:
The logFile isn't an option, its just defined beside the options - I can move it down. 
I droped container dir, I thought it was out-dated. I can put it back if need be.